site_scons/darwin.py

```diff
@@ -0,0 +1,52 @@
+from SCons.Script import *
+import sys
+import os
+
+
+
+def darwincompile(env):
+  print "darwincompile"
+  platform = sys.platform
+  "sets up environment for darwin, reading bash variables"
+  if (platform == 'darwin'):
+    # Set this to the SDK. XCode 7 needs MacOSX10.11.sdk, Xcode 6.x needs MacOSX10.10.sdk
+    # export OSX_SDK='/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk'
+    if 'OSX_SDK' in os.environ:
+      OSX_SDK = os.environ['OSX_SDK']
+    else:
+      OSX_SDK = '/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk'
```

Comment:
I cant test it on xcode 6.x, but this might be better: `xcodebuild -version -sdk macosx Path`